src/App.scss

```diff
@@ -1 +1,39 @@
-// style go here
+body {
+  margin: 0;
+  padding: 60px;
+  display: flex;
+  background-color: #dadada;
+}
+
+.article {
+  padding: 20px;
+  max-width: 1024px;
+  min-width: 600px;
+  font-weight: 400;
```

Comment:
That's a default value, no need to set it here, if you don't change it in parent components
```suggestion
```